src/components/Layouts/Header.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-toolbar class="toolbar">
+    <v-toolbar-title>
+      <router-link :to="{ name: 'home' }">StackOverflow</router-link>
+    </v-toolbar-title>
+
+    <v-spacer/>
+
+    <div class="search">
+      <v-text-field
+        label="Search"
+        solo
+        @input="(q) => updateValue(q)"
```

Comment:
you can write `@input="updateValue"` and it should work in the same way

Reply:
fixed